stardog/templates/monitoring/stardog-servicemonitor.yaml

```diff
@@ -13,11 +13,33 @@ spec:
     matchLabels:
       app.kubernetes.io/name: {{ include "stardog.name" . }}
   endpoints:
-  - port: metrics
+  - basicAuth:
+      password:
+        name: {{ include "stardog.fullname" . }}-user-passwords
+        key: admin
+      username:
+        # unless there is a cleaner way of doing this, the name "admin" must be stored in a secret:
+        name: {{ template "stardog.fullname" . }}-admin-username
+        key: adminusername
+    interval: 3s
+    path: /admin/status/prometheus
+    port: stardog
     interval: 3s
     scrapeTimeout: 3s
   jobLabel: app.kubernetes.io/instance
   podTargetLabels:
     - app.kubernetes.io/component
     - app.kubernetes.io/name
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "stardog.fullname" . }}-admin-username
+  labels:
+    app.kubernetes.io/name: {{ include "stardog.name" . }}
+    helm.sh/chart: {{ include "stardog.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+data:
```

Comment:
```suggestion
stringData:
  adminusername: admin
```

Reply:
I was just about to look up a way to do that -- Thanks!